src/test/java/uk/co/ramp/covid/simulation/ModelTest.java

```diff
@@ -183,6 +181,8 @@ public void testLockdown() {
             }
         }
         assertTrue("Unexpectedly fewer infections before lockdown", totInfDuringLockdown < totInfBeforeLockdown);
+        // TODO-CHECK: I'm unsure this is a correct test. If the lockdown is very effective there
+        //  are very low cases in the population afterwards

```

Comment:
The test fails for me on the assert at line 167. We should expect the total number of infections to be lower in the lockdown scenario.

Reply:
You're right. The assert at line 186 probably doesn't make sense

Reply:
Does it still fail for you on 167? I couldn't reproduce that (could be fixed when I fixed the movement).